op-guide/docker-compose.md

```diff
@@ -95,14 +94,13 @@ To customize the cluster, you can edit the `docker-compose.yml` file directly. I
 
     For PD, TiKV, TiDB and tidb-vision, you can build Docker images from GitHub source code or local files for development and testing.
 
-    - To build the image of a component from GitHub source code, you need to leave the `image` field empty and set `buildFrom` to `remote`.
-    - To build PD, TiKV or TiDB images from the locally compiled binary file, you need to leave the `image` field empty, set `buildFrom` to `local` and copy the compiled binary file to the corresponding `pd/bin/pd-server`, `tikv/bin/tikv-server`, `tidb/bin/tidb-server`.
-    - To build the tidb-vision image from local, you need to leave the `image` field empty, set `buildFrom` to `local` and copy the tidb-vision project to `tidb-vision/tidb-vision`.
+    - To build PD, TiKV or TiDB images from the locally compiled binary file, you need to leave the `image` field empty and copy the compiled binary file to the corresponding `pd/bin/pd-server`, `tikv/bin/tikv-server`, `tidb/bin/tidb-server`.
+    - To build the tidb-vision image from local, you need to leave the `image` field empty and copy the tidb-vision project to `tidb-vision/tidb-vision`.
```

Comment:
you need to **comment the `image` field** and copy the tidb-vision project to `tidb-vision/tidb-vision`.